packages/backend/src/routes/settings.routes.ts

```diff
@@ -1,11 +1,14 @@
 import { Router } from 'express';
 import { apiErrorHandler } from '../controllers/error/handler';
 import { setRefreshRateRequest, getRefreshRateRequest } from '../controllers/refreshRate/refreshRate';
+import { getCourseListRequest, setCourseRequest } from '../controllers/courseList/courseList';
 export const settingsRoutes = Router();
 
 // register routes
 settingsRoutes.get('/refreshRate', getRefreshRateRequest);
 settingsRoutes.put('/reFreshRate', setRefreshRateRequest);
```

Comment:
```suggestion
settingsRoutes.put('/refreshRate', setRefreshRateRequest);
```
So fresh ists jetzt au wieder ned :P